src/components/layouts/Layout.tsx

```diff
@@ -1,22 +1,49 @@
-// Libraries
-import { Container, Flex, Stack } from '@chakra-ui/react';
-import { FC } from 'react';
+import { Box, Container, Flex, Stack } from '@chakra-ui/react';
+import { FC, useContext } from 'react';
+import { useRouter } from 'next/router';
 
-// Components
 import { Header } from '../UI';
 import { Footer } from './Footer';
+import { DocsNav, MobileDocsNav } from '../UI/docs';
+
+// Context
+import { NavLinksContext } from '../../context';
 
 interface Props {
   children?: React.ReactNode;
 }
 
 export const Layout: FC<Props> = ({ children }) => {
+  const router = useRouter();
+  const { _navLinks } = useContext(NavLinksContext);
+
   return (
-    <Container maxW={{ base: 'full', md: 'container.2xl' }} my={{ base: 4, md: 7 }}>
-      {/* adding min-height & top margin to keep footer at the bottom of the page */}
-      <Flex direction='column' minH='calc(100vh - 3.5rem)'>
-        <Header />
+    <Container
+      maxW={{ base: 'full', md: 'container.2xl' }}
+      my={{ base: 4, md: 7 }}
+      overflow='visible'
+    >
+      <Box position='sticky' top={{ base: 3, md: 7 }} zIndex={9}>
+        <Box backdropFilter='blur(10px)' opacity={0.9}>
```

Comment:
@nloureiro I might suggest increasing the opacity here (which would make the scrolling content beneath even less visible)

90%:
<img width="381" alt="image" src="https://user-images.githubusercontent.com/54227730/230536537-0a0fd49c-f875-4e75-9ef9-da61523b7903.png">

98%:
<img width="408" alt="image" src="https://user-images.githubusercontent.com/54227730/230536494-fedf226f-36e4-4907-87e3-4ae087c2abb2.png">

I think it would help prevent text from clashing visually when there is overlap

Reply:
The `backdropFilter` does not appear to be displaying with the layer above it having an opaque background.

And seeing this on mobile when opening the search.
<img width="376" alt="image" src="https://user-images.githubusercontent.com/54227730/230537144-6fa95a34-8c5e-46af-8207-228ea3b500dd.png">
The search is fixed positioning which previously aligned with the same margin as above the navbar outline, but with this setup it's shifting by a couple pixels when you scroll down.